Configurations/windows-makefile.tmpl

```diff
@@ -495,7 +587,7 @@ EOF
      if ($srcs[0] =~ /\.asm$/) {
          return <<"EOF";
 $obj$objext: $deps
-	\$(AS) \$(ASFLAGS) \$(ASOUTFLAG)\$\@ $srcs
+	\$(AS) $asflags \$(ASOUTFLAG)\$\@ $srcs
```

Comment:
A bit of an overkill, but all right. I mean I can't imagine varying assembler flags...